libs/domain/content/src/models/content.model.ts

```diff
@@ -13,17 +13,30 @@ export interface ContentQualifier {
   type?: string;
   id?: string;
   query?: string;
+  tags?: string[] | string;
   entities?: ContentEntity[];
 }
 
-export interface ContentField {
-  [key: string]: unknown;
+export interface ContentMeta {
   id: string;
+  type: string;
+  name?: string;
 }
 
-export interface Content<T = Record<string, unknown>> {
-  fields: T & ContentField;
-  id: string;
-  type: string;
+/**
+ * @deprecated Since version 1.4. Will be removed.
+ */
+interface DeprecatedContent {
```

Comment:
I didn't get. This interface has been just added and already deprecated?

Reply:
We use this interface in Content type, it's for readability purpose rather then I deprecate each fields in the main one